### DISCUSSION TITLE
Barcode Scanning

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5400] [Help]
I want to scan the barcode and print the barcode number in the text box
![image](https://user-images.githubusercontent.com/24252615/105958127-450afa00-608b-11eb-9557-788da6e9a3f3.png)
 I want to add the possibility for the user to scan barcodes with the mobile camera when they are using the app on their mobiles.

**Comment by edwardch** [comment]
Search through previous issues and comments, Check #5046 and read through the discussion - maybe it could give you inspiration.

You will probably need some library to initiate the camera/barcode scanner. 
The first thing that came up in a search for me was https://serratus.github.io/quaggaJS/
I personally have not used this, I'm sure there are other libraries out there that do similar things!
